Rail/Address/internal.ts

```diff
@@ -1,12 +1,20 @@
 import * as cryptly from "cryptly"
+import { isly } from "isly"
 
 export interface Internal {
 	type: "internal"
+	name?: string
 	identifier: cryptly.Identifier
+	organization?: cryptly.Identifier
 }
 export namespace Internal {
 	export const currencies = ["EUR", "GBP", "SEK", "USD"] as const
-	export function is(value: Internal | any): value is Internal {
-		return typeof value == "object" && cryptly.Identifier.is(value.identifier, 8) && value.type == "internal"
-	}
+	export const type = isly.object<Internal>({
+		type: isly.string("internal"),
+		name: isly.string().optional(),
+		identifier: isly.fromIs("Identifier", cryptly.Identifier.is),
+		organization: isly.fromIs("Identifier", cryptly.Identifier.is).optional(),
```

Comment:
organization code is not necessarily a cryptly identifier

Reply:
:heavy_check_mark: